scripts/verify-package.sh

```diff
@@ -12,11 +12,78 @@ fi
 
 set -e
 
+# Internal packages to check for potential Dependency Confusion Attack (OKTA-529256)
+INTERNAL_PACKAGES_TO_FAKE=("@okta/typingdna" "@okta/handlebars-inline-precompile" "@okta/okta-i18n-bundles" "@okta/duo" "@okta/qtip")
+INTERNAL_PACKAGES=(${INTERNAL_PACKAGES_TO_FAKE[@]} "@okta/okta")
+inject_marker="f@@@@@@ke"
+
+# Inject fake packages with same names as ones that are used internally
+inject_fake_packages() {
```

Comment:
Please add a method to reverse this process and make sure that if we run this script locally it will reverse the changes even if the script fails. See the trap for "restore_node_modules" as an example.